docker/push

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
```

Comment:
can we use the same file of this from other repo, so that, we will have only one push file?

Reply:
still a few changes are required - that will make it reusable across the repos. for example, the master branch name is different on this repo, compared to the other one.